### ISSUE TITLE
A z-index problem when use popup widgets in table cell

### ISSUE THREAD

**Issue by bradwoo8621** [InsureMO/rainbow-d9#171]
- Use popup widget in table cell, such as dropdown, calendar, etc.
- Table has fix columns

When a column has dropdown widget, and it is scrolled to half hide under fix columns, click the dropdown. this cell's z-index will be reseted to greater than fix columns's, therefore the whole cell will in front of fix column.

**Current situation**
z-index of table widgets
- 1: expand area
- 2: row index cell / body cell stick in left
- 3: body cell stick in right / row operators
- 4: header cell
- 5: header cell stick in left
- 6: header cell stick in right
- 7: focus within in body cell, focus within in expand area

so when focus within met, it is on top of others, including fix columns.

**Predicament**

z-index of the active cell (focus within), need to be:
- Greater than or equals headers's, since popup might at to of cell,
- Less than left fixed columns, since it is under bottom of fix columns,
- Greater than other cells of unfixed columns, since popup might be covered by them.

Since cells of fix columns must under headers, since it might be scrolled. Obviously contradicts Target 1 and Target 2, making it impossible to achieve.

**Recommendation**

Don't use fix columns and dropdown widget in cell at the same time.

**Comment by bradwoo8621** [comment]
could be fixed, when DropdownUtils.findPortalCarrier is provided, this feature will be added at v1.2.5

**Comment by bradwoo8621** [comment]
fixed in `v1.2.8`, use the following settings on application startup.
make sure the `app-frame` dom node has all style variables which popup needs.

```ts
DropdownUtils.setDropdownDefaults({
    findPortalCarrier: () => document.getElementById('app-frame')!
});
```
